### DISCUSSION TITLE
unmodified control library is not needed for this pipeline?

### DISCUSSION THREAD

**Discussion by baibing7713661** [GoekeLab/m6anet#21] [Q&A] [answered]
Please correct me if I am wrong, this method is trained with the unmodified and modified direct RNA sequencing library. Therefore no unmodified library is needed for the direct RNA sequencing design when applying this pipeline to accurately detect the m6A site. Is that correct? If it's true, it will be really great. Is this method applied for the detection of m6A in RNA from any species such as plants and viruses?

**Comment by chrishendra93** [comment, marked as answer]
hi @baibing7713661 ,

Apology for the delay - I have replied to this discussion thread but it seems the message was not sent before. But to answer the question, yes, no unmodified library is needed as m6Anet is trained using labels to recognize m6A from a single sample of direct RNA sequencing data. I have not personally tried m6Anet on plants and viruses but in theory it should work as well. Recently there is a pre-print on biorxiv that did this for SARS-CoV-2 m6A sites https://www.biorxiv.org/content/biorxiv/early/2021/08/29/2021.08.24.457397.full.pdf
